src/docs/getting-started/installation.md

```diff
@@ -4,66 +4,40 @@ description: "How to install probe-rs."
 order: 20
 ---
 
+# Installing a precompiled binary (`cargo binstall`)
 
-# Prerequisites
+## Prerequisites
 
-To build probe-rs, you will need a working Rust toolchain. The easiest way is to follow the [rustup installation instructions](https://rustup.rs).
+First, set up `cargo-binstall`. You can find setup instructions [`here`](https://crates.io/crates/cargo-binstall).
```

Comment:
Don't we want to recommend installing via the bash script or brew? Think we should recommend the least effort.

Reply:
Is probe-rs available via brew?

Reply:
Yup :)

Reply:
really? fyi I couldn't find it on brew.sh so I removed the instr from here for now